cirq/protocols/json_test_data/Calibration.repr

```diff
@@ -1,13 +1,13 @@
 cirq.google.Calibration(
-    metric_dict={
+    metrics={
         'xeb': {
             (cirq.GridQubit(0, 0), cirq.GridQubit(0, 1)): [0.9999],
             (cirq.GridQubit(0, 0), cirq.GridQubit(1, 0)): [0.9998]
             },
         't1': {
-            (cirq.GridQubit(0, 0),): [321.0],
-            (cirq.GridQubit(0, 1),): [911.0],
-            (cirq.GridQubit(1, 0),): [505.0]},
-        'globalMetric': {(): [12300]}
+            (cirq.GridQubit(0, 0),): [321],
+            (cirq.GridQubit(0, 1),): [911],
+            (cirq.GridQubit(1, 0),): [505]},
+        'globalMetric': {(): ['abcd']}
```

Comment:
Consider adding a test for an unsupported type to assert the expected behavior (whatever it is).

Reply:
Done.